exchanges/binance/binance_wrapper.go

```diff
@@ -2026,22 +2020,14 @@ func (b *Binance) GetLatestFundingRates(ctx context.Context, r *fundingrate.Late
 			if !isPerp {
 				continue
 			}
-			var fundingRateFrequency int64
-			for x := range fri {
-				if fri[x].Symbol != mp[i].Symbol {
-					continue
-				}
-				fundingRateFrequency = fri[x].FundingIntervalHours
-				break
-			}
 			nft := time.UnixMilli(mp[i].NextFundingTime)
 			rate := fundingrate.LatestRateResponse{
 				TimeChecked: time.Now(),
 				Exchange:    b.Name,
 				Asset:       r.Asset,
 				Pair:        cp,
 				LatestRate: fundingrate.Rate{
-					Time: time.UnixMilli(mp[i].Time).Truncate(time.Hour * time.Duration(fundingRateFrequency)),
+					Time: time.UnixMilli(mp[i].Time).Truncate(time.Hour),
```

Comment:
```suggestion
					Time: time.UnixMilli(mp[i].NextFundingTime).Add(-time.Hour * 8),
```

As it just retrieves current time. And the ones below?

Reply:
Nah, binance has multiple funding rate times. On the upside, the endpoint is back online, so I guess I had a _KNEEJERK REACTION 💥_